### PR TITLE
Add snap option to allow disabling redis token setup

### PR DESCRIPTION
**Pull request by MonicaisHer**
close: https://github.com/canonical/edgex-ekuiper-snap/issues/18

### CODE REVIEW

**Review by farshidtz — CHANGES_REQUESTED** [review]
Thanks for the addition. It should do the job but in a very complicated way. The added go app increases the size of the snap but it's doing anything.

File: snap/hooks/configure [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+LOG_PREFIX=$SNAP_INSTANCE_NAME:configure
+EDGEX_SECURITY=$(snapctl get edgex-security)
+
+if [ "$EDGEX_SECURITY" != "off" ] && [ "$EDGEX_SECURITY" != "on" ] ; then
+	logger --stderr "$LOG_PREFIX: edgex-security can only be set to either 'on' or 'off'. When unset, it defaults to 'on'"
+  exit 1
+fi
```
Comment (farshidtz) [inline snap/hooks/configure:9]: @MonicaisHer I added some validation which I have not tested. Please check.

Comment (MonicaisHer) [reply snap/hooks/configure:9]: This works:
```
if [ "$EDGEX_SECURITY" == "" ] ; then
  :
elif [ "$EDGEX_SECURITY" != "off" ] && [ "$EDGEX_SECURITY" != "on" ] ; then
	logger --stderr "$LOG_PREFIX: edgex-security can only be set to either 'on' or 'off'. When unset, it defaults to 'on'"
  exit 1
fi
```

**Review by farshidtz — APPROVED** [review]
Thanks @MonicaisHer. It looks good. I've created the issue #21 to discuss how we can make this more configurable.

**Pull request merged**